src/main/java/com/ecommerce/ECommerceApp/model/Category.java

```diff
@@ -1,5 +1,9 @@
 package com.ecommerce.ECommerceApp.model;
 
+/**
+ * Enumerator containing product categories and subcategories.
+ * Subcategories include: ELECTRONICS, FURNITURE, and CLOTHING
+ */
```

Comment:
Added missing Javadoc to help clarify purpose of code, keeping it maintainable and readable.

Source: https://www.tutorialspoint.com/java/java_documentation.htm